iso-19115-19119/ds-specification.md

```diff
@@ -10,7 +10,7 @@ The test first checks if there is at least one [specification](#specification).
 It then performs the following checks
 *	The [specification](#specification) must contain an element of type gmd:CI_Citation/gmd:title which should not be an [empty characterstring](http://inspire.ec.europa.eu/id/ats/metadata/1.3/iso-19115-19119/README#emptychar)
 *	The [specification](#specification) must contain an element of type gmd:CI_Citation/gmd:date[./\*/gmd:dateType/\*/text()='{type}']/\*/gmd:date, where {type} is one of 'creation', 'revision' and 'publication'.
-*	The [specification](#specification) has gmd:DQ_DomainConsistency as a parent element.
+*	Metadata have at least one [DQ_ConformanceResult](#DQ_ConformanceResult) element.
```

Comment:
I think this is wrong, and the original text is (kind of) correct. But the test (and the original requirement) are not formulated well. It should be similar to the test for req. 28 (which also needs to be revised - see below). Something like this:

This test case only applies to records with a hierarchyLevel value 'dataset' or 'series'.

For every gmd:report/gmd:DQ_DomainConsistency/gmd:result/gmd:DQ_ConformanceResult element, check if there is at least one specification. In case there is none, the test fails. If there is one, perform the following checks

* The [specification](#specification) must contain an element of type gmd:CI_Citation/gmd:title which should not be an [empty characterstring](http://inspire.ec.europa.eu/id/ats/metadata/1.3/iso-19115-19119/README#emptychar)
* The [specification](#specification) must contain an element of type gmd:CI_Citation/gmd:date[./\*/gmd:dateType/\*/text()='{type}']/\*/gmd:date, where {type} is one of 'creation', 'revision' and 'publication'.
* The gmd:DQ_ConformanceResult has an element gmd:pass that must contain a value of type gco:Boolean.

In addition, a fix should be implemented on ds-conformity.md, which should check  if there is at least a gmd:report/gmd:DQ_DomainConsistency/gmd:result/gmd:DQ_ConformanceResult element citing to the INSPIRE implementing rules on interoperability of spatial data sets and services, according to the rules for specification and date elements specified in req. 29, i.e. with the same logic but concrete values for specification and date.

Reply:
Thanks for the comments, the test have been modified according to you explanations. 

We would like if you can help us with some doubs regarding the modifications under ds-conformity.md. You mention that we should implement the same logic the test before, but also checking concrete values for specification and date.After reviewing the TG, we did not found which concrete values are the requiered. Could you please detail us those concrete values?

This is the only pending modification for this Helpdesk pull request.

Reply:
Please use the date, date type and title from this CI_Citation element (taken from v2.0 of the TG):

```xml
<gmd:CI_Citation>
   <gmd:title>
     <gmx:Anchor xlink:href="http://data.europa.eu/eli/reg/2010/1089">COMMISSION REGULATION (EU) No 1089/2010 of 23 November 2010 implementing Directive 2007/2/EC of the European Parliament and of the Council as regards interoperability of spatial data sets and services</gmx:Anchor>
   </gmd:title>
   <gmd:date>
     <gmd:CI_Date>
       <gmd:date>
         <gco:Date>2010-12-08</gco:Date>
       </gmd:date>
       <gmd:dateType>
         <gmd:CI_DateTypeCode codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#gmxCodelists.xml#CI_DateTypeCode"                 codeListValue="publication">publication</gmd:CI_DateTypeCode>
       </gmd:dateType>
     </gmd:CI_Date>
   </gmd:date>
</gmd:CI_Citation> 
```

Reply:
As discussed in the meeting on 9/8/18, the title text in the CI_Citation should be the title in the relevant MD language. The correct titles in the different languages are available in the language-specific files at http://inspire.ec.europa.eu/schemas/common/1.0/enums/

Reply:
Dear @michellutz, could you please verify that the titles must be exactly the same that in the provided list. In some examples the words don't match because of uppercase and lowercase letters.

Thanks.

Reply:
I would suggest that the strings should match, allowing for different capitalisation of words and also additional spaces, i.e. the following should all be a match for the normative text "COMMISSION REGULATION (EU) No 1089/2010 of 23 November 2010 implementing Directive 2007/2/EC of the European Parliament and of the Council as regards interoperability of spatial data sets and services":
- Commission Regulation (EU) No 1089/2010 of 23 November 2010 implementing Directive 2007/2/EC of the European Parliament and of the Council as regards interoperability of spatial data sets and services
- Commission regulation (EU) No 1089/2010 of 23 November 2010 implementing Directive 2007/2/EC of the European Parliament and of the Council as regards Interoperability of Spatial Data Sets  and  Services [note the additional spaces]

If this is difficult to implement, I would suggest simply checking two versions starting with "COMMISSION REGULATION ..." and "Commission Regulation ..."